actiontext/app/helpers/action_text/content_helper.rb

```diff
@@ -22,13 +22,25 @@ def render_action_text_attachments(content)
       content.render_attachments do |attachment|
         unless attachment.in?(content.gallery_attachments)
           attachment.node.tap do |node|
-            node.inner_html = render(attachment, in_gallery: false).chomp
+            partial = if attachment.respond_to?(:to_attachable_partial_path)
+              attachment.to_attachable_partial_path
```

Comment:
@georgeclaghorn I've given a customizable partial path my best shot.

Is `#to_attachable_partial_path` an acceptable method name? I've named it after the `ActionText::Attachable` concern, but that module name might be considered a private API.

I'm open to suggestions.